specs/phase0/p2p-interface.md

```diff
@@ -221,12 +220,12 @@ The payload is carried in the `data` field of a gossipsub message, and varies de
 
 | Name                                           | Message Type            |
 |------------------------------------------------|-------------------------|
-| beacon_block                                   | SignedBeaconBlock       |
-| beacon_aggregate_and_proof                     | SignedAggregateAndProof |
-| committee_index{subnet_id}\_beacon_attestation | Attestation             |
-| voluntary_exit                                 | SignedVoluntaryExit     |
-| proposer_slashing                              | ProposerSlashing        |
-| attester_slashing                              | AttesterSlashing        |
+| beacon\_block                                  | SignedBeaconBlock       |
+| beacon\_aggregate\_and\_proof                  | SignedAggregateAndProof |
+| beacon_attestation\_{subnet\_id}               | Attestation             |
+| voluntary\_exit                                | SignedVoluntaryExit     |
+| proposer\_slashing                             | ProposerSlashing        |
+| attester\_slashing                             | AttesterSlashing        |
```

Comment:
```suggestion
| Name                             | Message Type              |
|----------------------------------|---------------------------|
| `beacon_block`                   | `SignedBeaconBlock`       |
| `beacon_aggregate_and_proof`     | `SignedAggregateAndProof` |
| `beacon_attestation_{subnet_id}` | `Attestation`             |
| `voluntary_exit`                 | `SignedVoluntaryExit`     |
| `proposer_slashing`              | `ProposerSlashing`        |
| `attester_slashing`              | `AttesterSlashing`        |
```

Instead of the weird escaping, we should just use code here, like in the text and the table above.